src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/ScannerOffJobProperty.java

```diff
@@ -36,7 +36,7 @@
  * @deprecated {@link ScannerJobProperty} is used instead, but this is kept to be able to de-serialize old jobs.
  */
 @Deprecated
-public class ScannerOffJobProperty extends JobProperty<AbstractProject<?, ?>> implements Serializable {
+public class ScannerOffJobProperty extends JobProperty<Job<?, ?>> implements Serializable {
```

Comment:
You didn't need to change this one, it only exists for older jobs and is deserialized as a `ScannerJobProperty`